sykmeldingalerts.yaml

```diff
@@ -51,8 +51,20 @@ spec:
       action: "Sjekk hvorfor {{ $labels.log_app }} har i løpet av dei siste 5 minuttene logget Error"
       sla: respond within 1h, during office hours
       severity: danger
+    - alert: syfo-errorlogging
```

Comment:
Vi har litt høyere terskel antall feil i loggen før vi fyrer alert for denne appen enn for sm-appene, derfor lagde jeg en egen alert for varsel :)